Zend/zend_vm_execute.skl

```diff
@@ -4,6 +4,7 @@
 # pragma GCC push_options
 # pragma GCC optimize("no-gcse")
 # pragma GCC optimize("no-ivopts")
+# pragma GCC optimize("no-reorder-blocks-and-partition")
```

Comment:
The flag ``-fno-reorder-blocks-and-partition`` should be passed to CC through ``CFLAGS`` in Makefile.
- Using this line here won't disable this optimization for other files.
- This disables useful optimization for